index.js

```diff
@@ -338,7 +338,7 @@ module.exports = {
     'no-var': 2,
     // 'object-shorthand': 0,
     // 'prefer-arrow-callback': 0,
-    // 'prefer-const': 0,
+    'prefer-const': ['error', {'destructuring': 'all'}],
```

Comment:
Sorry, should have also mentioned, can you remove the quotes from the object key `destructuring`? It should be:

```js
'prefer-const': ['error', {destructuring: 'all'}],
```

Reply:
You know, I have "allow edits from maintainers" turned on ;)

Reply:
I did not know that! But it all looks good now anyway, thanks!